### ISSUE TITLE
HELP! My character is semi-transparent and other models are shadow-people!

### ISSUE THREAD

**Issue by broxen** [Segs/Segs#736]
## Summary
`HELP! My character is semi-transparent and other models are shadow-people!`

This is a known issue with Intel or AMD/Radeon graphics cards. The client version we use is not capable of handling those graphics cards. I created this issue so that users encountering this bug can find the current proposed solution: _please wait until the client patch is completed._

A patch is in the works #450 but we need a skilled debugger that's capable of testing/debugging on an Intel or Radeon card. If you're experienced in debugging on Windows please reply here, or in #450

**Comment by swisscub** [comment]
I'm unfortunately not a debugger nor am I experienced in it, but I hope there will be a solution soon, as I'm one of the users with an AMD gfx card. Thanks in advance! :)

**Comment by JamesDW1** [comment]
Managed to make the client patch run with the client after a long time of the client closing out after the Loading bar. I was able to load into the game fine, however it seemed as though I was unable to transfer zones. Loading into the zones appeared to work fine, as I managed to enter Atlas Park through editing the db; however, transferring maps lead to a client crash with the Cryptic crash report box showing up. I am unable to provide pictures as it now seems that the earlier behavior of the client closing out after the Loading bar has returned. I thought it was worth noting this here.
